runtime/parachains/src/runtime_api_impl/mod.rs

```diff
@@ -21,3 +21,4 @@
 //! to a v2 would be done.
 
 pub mod v2;
+pub mod vstaging;
```

Comment:
This makes sense. Thanks

Reply:
We might put this under a feature flag `vstaging` and link with `--no-default-features` from the production runtimes.

Reply:
We definitely can do this if it is okay to return `unimplemented!()` (what you referred in [this comment](https://github.com/paritytech/polkadot/pull/5048#discussion_r846838848)).